### ISSUE TITLE
Online validator is unstable

### ISSUE THREAD

**Issue by Alexander-Barth** [INSPIRE-MIF/helpdesk-validator#237]
I get the attached error messages when using the online validator. No report is produced.

- [x] a description of the steps to reproduce your issue

* go to http://inspire.ec.europa.eu/validator/#start-tests
* choose WMS
* start test
* insert the address: https://ec.oceanbrowser.net/test-1/Python/web/wms?REQUEST=GetCapabilities&SERVICE=WMS&VERSION=1.3.0

- [x] the browser and version used

Firefox 70.0.1 and google-chrome (80.0.3987.122-1)

- [x] a screenshot illustrating the problem

Error: Connection to test runner lost. The test could have been canceled without generating  a report or the internet connection to the web application has been lost!

![Bildschirmfoto von 2020-03-04 10-49-07](https://user-images.githubusercontent.com/9881475/75870115-d08ff280-5e0a-11ea-99f8-4465d7a5cede.png)

![Bildschirmfoto von 2020-03-02 17-07-41_crop](https://user-images.githubusercontent.com/9881475/75870326-21075000-5e0b-11ea-88bb-41cae041fbec.png)

![Bildschirmfoto von 2020-03-02 14-35-33](https://user-images.githubusercontent.com/9881475/75870614-83605080-5e0b-11ea-80a4-6043792be820.png)

- [ ] the Test Report (ideally as an attachment to the issue)

no report is generated

**Comment by danielnavarrogeo** [comment]
Dear @Alexander-Barth 

We have try to reproduce your error but we could not. It seems to be due to heavy load in the Validator when you perform the validation.

We are going to search for the causes of this issue.

Regards

**Comment by carlospzurita** [comment]
Dear @Alexander-Barth 

We are not able to reproduce the error, neither in the production environment or in a local one. We only found an error in the logs related to the execution workflow of the TestRun. 

`2020-03-04 09:48:15.550 [TestRun.887f56f0-b400-4e8d-8c07-253e6bb8350b] INFO  d.i.e.t.s.STestCaseRunner - Project [Conformance Class: View Service - WMS] finished with status [FAILED] in 10756ms
2020-03-04 09:48:15.565 [TestRun.887f56f0-b400-4e8d-8c07-253e6bb8350b] ERROR d.i.e.t.DefaultTestRun - Illegal state transition in task 887f56f0-b400-4e8d-8c07-253e6bb8350b from COMPLETED to FAILED`

It seems the TestRun finished and generated a result, but it tried to restart it. We are testing it again but it is working fine now.

Please check again on your side, and if there are no errors, we will close this issue for now.

**Comment by Alexander-Barth** [comment]
I have used the validator 6 time since then and it worked 4 times. Once I got an spinner rotating endlessly and the other time I got a "504 Gateway Time-out" (I can provide screenshots, but I don't know if this is really useful, but I pasted the time stamps below). Subjectively, the success rate is feels now higher when I first tried, but I would say, there is still room.

These are the time stamps (in CET)
2020-03-12 09-29-32 -> spinner issue
2020-03-18 09-21-12 -> 504 Gateway Time-out
